sunpy/net/jsoc/jsoc.py

```diff
@@ -524,11 +524,13 @@ def get_request(self, requests, path=None, overwrite=False, progress=True,
             dl_set = False
             downloader = Downloader(progress=progress, overwrite=overwrite, max_conn=max_conn)
 
-        if downloader.max_conn * kwargs['max_splits'] > 10:
-            warn_user("JSOC does not support more than 10 parallel connections. " +
-                      f"Changing the number of parallel connections to {2 * self.default_max_conn}.")
-            kwargs['max_splits'] = 2
-            downloader.max_conn = self.default_max_conn
+        # Private communication from JSOC say we should not use more than one connection.
+        if downloader.max_conn * kwargs['max_splits'] > 1:
+            warn_user(
+                "JSOC does not support more than 1 parallel connection, changing the number to 1."
+            )
+            kwargs['max_splits'] = 1
+            downloader.max_conn = 1
```

Comment:
This is a problem, as it affects downloads from all clients using this downloader not just JSOC.

I am pro setting the splits to 1, I am less keen about doing this.

Reply:
Changed.